concept/ConceptManager.java

```diff
@@ -187,8 +187,4 @@ public GraknException exception(ErrorMessage error) {
     public GraknException exception(Exception exception) {
         return graphMgr.exception(exception);
     }
-
-    public GraknException exception(GraknException exception) {
```

Comment:
Removed `exception(GraknException)` method and only keeps `exception(Exception)` so that we only need to determine if we want to convert exception to `GraknException` at the lowest level.